public/app/plugins/datasource/grafana/types.ts

```diff
@@ -27,9 +29,14 @@ export interface GrafanaQuery extends DataQuery {
   path?: string; // for list and read
   search?: SearchQuery;
   snapshot?: DataFrameJSON[];
+  timeRegion?: TimeRegionConfig;
   file?: GrafanaQueryFile;
 }
 
+export interface TimeRegionConfig extends BaseTimeRegionConfig {
+  timezone?: string;
```

Comment:
Lets just put timezone on the original `TimeRegionConfig` and avoid the extension (not needed now that it is not a full query with styling etc)